azure-pipelines.yml

```diff
@@ -49,9 +49,33 @@ jobs:
       initSteps:
         - pwsh: |
             dotnet workload update --verbosity diag --from-rollback-file $(XamarinDotNetWorkloadSource) --source $(Dotnet6Source) --source $(NuGetOrgSource)
+            if ($LASTEXITCODE -ne 0) {
+                Write-Host "##vso[task.logissue type=error]Failed to set up workload rollback file."
+                Write-Host "##vso[task.complete result=Failed;]"
+                exit 0
+            }
+            
             dotnet workload install android --verbosity diag --skip-manifest-update --source $(Dotnet6Source) --source $(NuGetOrgSource)
```

Comment:
Can we delete this line? The `maui` workload extends `android`, so it will install it.